Patches/YAMLcpp/CMakeLists.txt

```diff
@@ -179,9 +179,7 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR
 	#
 	set(GCC_EXTRA_OPTIONS "")
 	#
-	if(BUILD_SHARED_LIBS)
-		set(GCC_EXTRA_OPTIONS "${GCC_EXTRA_OPTIONS} -fPIC")
-	endif()
+	set(GCC_EXTRA_OPTIONS "${GCC_EXTRA_OPTIONS} -fPIC")
```

Comment:
The proper, modern way to do this is 
`set(CMAKE_POSITION_INDEPENDENT_CODE ON)`
which shouldn't require editing of the CMake file. Though that is an option, it might make it more apparent if we add 
`-DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON`
as a CMake argument in CMake/External_YAMLcpp.cmake. That will be less mysterious for a future user who is wondering where it came from.

Reply:
Ah, I see.  I've pushed up changes to this effect